esx_service/vmdk_ops_test.py

```diff
@@ -1082,6 +1084,87 @@ def test_vmdkops_on_different_tenants(self):
         self.assertEqual("tenant2_vol1", result[0]['Name'])
         self.assertEqual("tenant2_vol2", result[1]['Name'])
         self.assertEqual("tenant2_vol3", result[2]['Name'])
+    
+    @unittest.skipIf(len(vmdk_utils.get_datastores()) < 2,
+                     "second datastore is not found - skipping this test")
```

Comment:
I think that If we need 2 DS for a test, then we need to request it and make sure test bed has 2 datastores - instead of skipping the test. 
@kerneltime  - comments ?